src/main/webapp/script.js

```diff
@@ -167,13 +174,24 @@ async function addToProfile(route) {
     });
 }
 
-function createCommentsPanel(route) {
+function generateRating(route) {
+  let ratingElement = document.createElement("p");
+  ratingElement.classList.add("route-rating");
+
+  console.log(route.sumOfRatings);
+  console.log(route.numberOfRatings);
+
+  let rating = route.sumOfRatings / route.numberOfRatings;
```

Comment:
for greater security i think it may be useful to check if numberOfRatings not a 0.

Reply:
Done